src/test/java/eu/webeid/security/validator/AuthTokenCertificateTest.java

```diff
@@ -229,6 +229,7 @@ void whenUserCertificateIsNotYetValid_thenValidationFails() {
     }
 
     @Test
+    @Disabled("It is no longer feasible to check if trusted CA certificate is valid as subject certificate validity is checked before")
```

Comment:
Why not remove the test then?

Reply:
Thanks, fair point. I modified and re-enabled the tests and added comments why they are still relevant. Also added `whenUnrelatedCACertificateIsExpired_thenValidationSucceeds()` explicitly for this case.